packages/react/src/internal/useNormalizedInputProps.js

```diff
@@ -44,6 +44,7 @@ import { usePrefix } from './usePrefix';
  * @param {InputProps} props - The props passed to the component
  * @returns {NormalizedInputProps}
  */
+/** @type any */
```

Comment:
you can probably get rid of this by documenting the `helperId` property in `NormalizedInputProps` above; something like `* @property {string} helperId - id used for helper text` should work